src/core/style/layout.scss

```diff
@@ -43,6 +43,13 @@
   min-height: 100%;
   width: 100%;
 }
+@-moz-document url-prefix() { 
+  [layout-fill] {
+    margin: 0;
+    width: 100%;
+    min-height: auto;
```

Comment:
I'm not sure this does as intended. In FF, fill-layout no longer makes a container full height, and overflow breaks in e.g. md-content. Works in Chrome.